include/arch/arm/error.h

```diff
@@ -53,6 +53,11 @@ extern "C" {
 		: "memory"); \
 	CODE_UNREACHABLE; \
 } while (false)
+#elif defined(CONFIG_ARMV7_R)
+#define Z_ARCH_EXCEPT(reason_p) do { \
+	k_sys_fatal_error_handler(K_ERR_KERNEL_PANIC, NULL); \
```

Comment:
you should not be invoking k_sys_fatal_error_handler() directly! for one thing, it can return.

either leave this undefined so that the default definition in include/kernel.h is used, or better yet get the kernel oops code in the SVC handler working on this arch

Reply:
I let it pick up the default definition for now.  I will fix it to use the svc handler in a future PR.